src/amo/components/EditableCollectionAddon/styles.scss

```diff
@@ -29,6 +29,10 @@
   display: inline-block;
   height: 32px;
   width: 32px;
+
+  &.EditableCollectionAddon-icon--statictheme {
+    opacity: 0;
```

Comment:
well, I thought about that but I thought we wanted the white space (just waiting to hear from Philip), b/c then it would look like this:

![Alt text](https://monosnap.com/image/kyMsjp7XUPgPiMoPbPYihu3K4TBhFt.png)

so I kept it for styling purposes...

Reply:
ping @pwalm

Reply:
hey @willdurand, thanks for the feedback. given this design is what @pwalm wants...is it ok to use the opacity or would you still rather see the img tag removed for static themes and restyle the SearchSuggestion-name?